src/main/java/org/logstash/beats/Server.java

```diff
@@ -110,15 +110,15 @@ public boolean isSslEnable() {
 
         private final EventExecutorGroup idleExecutorGroup;
         private final EventExecutorGroup beatsHandlerExecutorGroup;
-        private final IMessageListener message;
-        private int clientInactivityTimeoutSeconds;
-
-        private boolean enableSSL = false;
-
-        public BeatsInitializer(Boolean secure, IMessageListener messageListener, int clientInactivityTimeoutSeconds, int beatsHandlerThread) {
-            enableSSL = secure;
-            this.message = messageListener;
-            this.clientInactivityTimeoutSeconds = clientInactivityTimeoutSeconds;
+        private final IMessageListener localMessageListener;
+        private final int localClientInactivityTimeoutSeconds;
+        private boolean localEnableSSL;
```

Comment:
This can be `final` as it is initialized in the constructor

Reply:
Indeed, I missed this one.